launcher/launch/LogModel.h

```diff
@@ -32,12 +32,12 @@ class LogModel : public QAbstractListModel {
 
    private /* types */:
     struct entry {
-        MessageLevel::Enum level;
-        QString line;
+        MessageLevel::Enum level = MessageLevel::Enum::Unknown;
+        QString line = {};
     };
 
    private: /* data */
-    QVector<entry> m_content;
+    QVector<entry> m_content = {};
```

Comment:
Why would this be undefined behaviour

Reply:
why whould `bool x;` be undefined?
I just initiated all the class variables in the places where the undefined behavior was reported.
Is it incorrect that I initialized it?

Reply:
Still, is it wrong that I initiated it, or is it just you nitpicking this exact thing?
Do you have a guideline that says that initializing it is wrong and I should not do it?
(Please do suggest any changes to this PR with "add suggestion")